.pre-commit-config.yaml

```
@@ -30,7 +30,7 @@ repos:
       # Run the formatter.
       - id: ruff-format
   - repo: https://github.com/PyCQA/docformatter
-    rev: eb1df34 # TODO: wait for https://github.com/PyCQA/docformatter/issues/293
+    rev: v1.7.5 # TODO: wait for https://github.com/PyCQA/docformatter/issues/293
```

```suggestion
    rev: eb1df34 # TODO: wait for https://github.com/PyCQA/docformatter/issues/293
```
add another ping to https://github.com/PyCQA/docformatter/issues/293